src/org/etools/j1939_84/utils/StringUtils.java

```diff
@@ -25,6 +25,20 @@ public static boolean containsNonPrintableAsciiCharacter(String string) {
         return NON_PRINTABLE_PATTERN.matcher(string).matches();
     }
 
+    /*
+     * @param string String to be checked for non-printable ASCII characters
+     */
+    public static boolean containsNonPrintableAsciiCharacter(byte[] bytes) {
+        boolean containsNonPrintableChar = false;
+
+        for (int i = 0; i < bytes.length; i++) {
+            if ((bytes[i] > 32) || (bytes[i] < 127)) {
+                containsNonPrintableChar = true;
```

Comment:
This can “return true” here.  The end of the method can then just “return false”

Reply:
#8 https://github.com/SolidDesignNet/j1939-84/pull/9/commits/f100a3abf4bfa4e33e1017f6a3581d4681a2e920